tools/mentors/view_page_text_image.php

```diff
@@ -206,7 +206,7 @@
 <form method="get" action="view_page_text_image.php">
 <input type="hidden" name="projectid" value="<?php echo $projectid; ?>">
 <input type="hidden" name="page" value="<?php echo $page; ?>">
-<input type="text" maxlength="3" name="percent" size="3" value="<?php echo $percent; ?>">%
+<input type="number" name="percent" value="<?php echo $percent; ?>" min="1" max="100">%
```

Comment:
Might be a good idea to add `required`.

Does not handle `5.0` and `5e0` well but I do not think it is much a concern here.

Reply:
https://github.com/DistributedProofreaders/dproofreaders/blob/a4f9aa136899dfd0b9ce52519e203be257c1c326/tools/mentors/view_page_text_image.php#L127

`page` could be a `number` as well.

Reply:
Actually, I don't think it can. Page "numbers" are really strings because they can contain more than just numbers (and also may have leading zeros).